source/server/controller/DBManager.php

```diff
@@ -2108,6 +2108,14 @@ public function deleteGames($ids)
                     gameid = ?"
             );
             $this->executeGameDeleteStatement($stmt, $ids);
+
+            $stmt = $this->connection->prepare(
+                "DELETE FROM 
+                    tac_flightsize
+                WHERE
+                    gameid = ?"
+            );
+            $this->executeGameDeleteStatement($stmt, $ids);
```

Comment:
This data was not deleted when a game was deleted and caused weird behaviour locally